sqlmesh/dbt/target.py

```diff
@@ -401,14 +401,13 @@ class RedshiftConfig(TargetConfig):
     sslmode: t.Optional[str] = None
 
     @model_validator(mode="before")
-    @model_validator_v1_args
-    def validate_database(
-        cls, values: t.Dict[str, t.Union[t.Tuple[str, ...], t.Optional[str], t.Dict[str, t.Any]]]
-    ) -> t.Dict[str, t.Union[t.Tuple[str, ...], t.Optional[str], t.Dict[str, t.Any]]]:
-        values["database"] = values.get("database") or values.get("dbname")
-        if not values["database"]:
-            raise ConfigError("Either database or dbname must be set")
-        return values
+    @classmethod
+    def validate_database(cls, data: t.Any) -> t.Any:
+        if isinstance(data, dict):
```

Comment:
Let's follow the same style of early return

Reply:
Yep, done everywhere